### PR TITLE
Minor changes to the sql API

### PR DESCRIPTION
**Pull request by vivekmenezes**

### CODE REVIEW

**Comment by petermattis** [comment]
LGTM

File: sql/sqlwire/sql_api.proto [hunk 1]
```diff
@@ -29,9 +29,11 @@ option (gogoproto.unmarshaler_all) = true;
 
 // SQLRequestHeader is supplied with every CmdRequest.
 message SQLRequestHeader {
+  // User is the originating user.
+  optional string user = 5 [(gogoproto.nullable) = false];
   // Session settings that were returned in the last response that
   // contained them, being reflected back to the server.
-  optional bytes session = 1;
+  optional bytes settings = 1;
```
Comment (bdarnell) [inline sql/sqlwire/sql_api.proto]: I like the name `session` better than `settings`. `settings` implies to me that it's under the caller's full control; `session` sounds like a better match for the cookie-like echoing done here.

**Comment by bdarnell** [comment]
LGTM

File: sql/sqlwire/sql_api.proto (hunk 1 above)
Comment (vivekmenezes) [inline sql/sqlwire/sql_api.proto]: sounds fine. I'll update both request/respone to session

File: sql/sqlwire/sql_api.proto [hunk 2]
```diff
@@ -46,7 +48,7 @@ message SQLResponseHeader {
   optional proto.Error error = 1;
   // Setting that should be reflected back in all subsequent requests.
   // When not set, future requests should continue to use existing settings.
-  optional bytes settings = 2;
+  optional bytes session = 2;
```
Comment (spencerkimball) [inline sql/sqlwire/sql_api.proto]: Update the comment which starts with "Setting".

**Pull request merged**